README.md

```diff
@@ -44,6 +44,28 @@ When launching the app, you can specify the port to use with a command like the
 ```
 ./gradlew bootRun -Pport=8888
 ```
+### Configuring the settings
+There are three ways for user to configure the settings. 
```

Comment:
Remove the sentence at line 48 and structure the rest of the paragraph like this:
1. When running the application from sources:
Change the file(as a link) directly or put a customized version of the file to overwrite the properties and put it under config folder before running the application.
Notes: all configurable properties are available in the example version of application.properties.
2. When running the application as .war file:
2.1 Create/Overwrite the file in config folder.
2.2 Overwrite specific properties as command line arguments.
<example>
Notes: Property column in configuration tables in this readme will be the related property name in application.properties.